modules/jpa/pom.xml

```diff
@@ -19,6 +19,18 @@
       <groupId>io.oasp.java.modules</groupId>
       <artifactId>oasp4j-jpa-dao</artifactId>
     </dependency>
+    <dependency>
+     <groupId>io.oasp.java.modules</groupId>
+  <artifactId>oasp4j-jpa-envers</artifactId>
```

Comment:
why do we need `oasp4j-jpa-envers` here? Wasn't the dependency other-way round before?

Reply:
please see comments given to AbstractGenericRevisionedDao